src/pages/waiting-room-to-car/cat-a-mod2/components/vehicle-checks/vehicle-checks.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+import { CAT_A_MOD2 } from '../../../../page-names.constants';
+import { ModalController } from 'ionic-angular';
+import { App } from '../../../../../app/app.component';
+import { VehicleChecksScore } from '../../../../../shared/models/vehicle-checks-score.model';
+
+// TODO - PREP-AMOD2: Use a mod2 types
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { get } from 'lodash';
+import { QuestionResult } from '@dvsa/mes-test-schema/categories/common';
+
+@Component({
+  selector: 'vehicle-checks-cat-a-mod2',
+  templateUrl: 'vehicle-checks.html',
```

Comment:
Is this correct? Is `vehicle-checks.html` shared throughout with the only difference being the selector?

Reply:
I think the selector shouldn't have the category in it, unless we want to include this alongside with another vehicle-checks to the same module. Would be nice to get to the bottom of this. But for consistency we can leave it as it is for now

Reply:
Cheers for the clarification :)